src/main/java/com/blazemeter/jmeter/http2/sampler/gui/HTTP2RequestPanel.java

```diff
@@ -14,16 +14,27 @@
 import org.apache.jmeter.testelement.property.JMeterProperty;
 import org.apache.jmeter.testelement.property.TestElementProperty;
 import org.apache.jmeter.util.JMeterUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.blazemeter.jmeter.http2.sampler.HTTP2Request;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ResourceBundle;
+
 import javax.swing.*;
 
 public class HTTP2RequestPanel extends JPanel {
 
     private static final int TAB_PARAMETERS = 0;
 
     private HTTPArgumentsPanel argsPanel;
+    
+    private static Logger log = LoggerFactory.getLogger(HTTP2RequestPanel.class);
```

Comment:
What do you think if we change it to add `final` since we are not modifying it, right?
Also, what do you think if we write the name in upper case? Like `LOG`?